plutus-context-builder/src/Test/Plutus/ContextBuilder.hs

```diff
@@ -141,407 +155,487 @@ import Test.Plutus.ContextBuilder.Minting (
  )
 import Wallet.Emulator.Types (Wallet, mockWalletPaymentPubKeyHash)
 
-{- | Single 'SideUTXO'-input context.
+{- | Anonymous context from a single 'SideUTXO' input.
 
- @since 1.0
+ @since 2.0
 -}
 input ::
   forall (p :: Purpose).
-  Text ->
   SideUTXO ->
-  ContextBuilder p
-input name x = mempty {cbInputs = Map.singleton name x}
+  ContextBuilder p 'Anonymous
+input x = NoNames $ mempty {cfInputs = pure x}
 
-{- | Single 'ValidatorUTXO'-input context.
+{- | Anonymous context from a single 'ValidatorUTXO' input.
 
  = Note
 
- This input won't be used as 'Spending' in the 'ScriptPurpose'
- of the builded 'ScriptContext'.
+ This input won't be used for spending in any 'ScriptPurpose' of any
+ 'ScriptContext' built from this.
 
- @since 1.0
+ @since 2.0
 -}
 validatorInput ::
   forall (d :: Type) (r :: Type).
   (FromData d, ToData d, Show d) =>
+  -- | Name of the input
```

Comment:
I'm not sure. @sergesku - do you have some insights?

Reply:
Yes, the purpose was exactly the same.

Reply:
@blamario Do we want to do to them the same as what we did to `ContextBuilder`s? I'm in favour of leaving them alone to avoid even more breaks, but if you'd rather be consistent, I can understand.

Reply:
I'm in favour of a consistent design as well.